event-handler/teleport_events_watcher.go

```diff
@@ -97,6 +103,31 @@ func NewTeleportEventsWatcher(
 	return &tc, nil
 }
 
+// validateConfig validates the given StartCmdConfig
+func validateConfig(cfg StartCmdConfig) error {
+	errs := []error{}
+	// Check all config fields with the tag existingfile point to an existing file.
+	for _, config := range []interface{}{cfg.FluentdConfig, cfg.TeleportConfig, cfg.IngestConfig} {
```

Comment:
Not sure I understand, why do we need to use reflection here instead of checking that these fields aren't empty?

Reply:
Was thinking it may be handy if any more eixistingfile fields were added, in hindsight this was kind of made no sense. Will just remove the reflection and have a check for each field.